### PR TITLE
Add Hjson recipe

### PR DESCRIPTION
**Pull request by nickpeihl**
This PR is a recipe for the [Hjson](http://hjson.org/) Python library.

### CODE REVIEW

File: recipes/hjson/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/hjson/meta.yaml]: Please remove comments.

File: recipes/hjson/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
```
Comment (synapticarbors) [inline recipes/hjson/meta.yaml]: You might consider testing the command line tools as well. I have a memory of this not being handled well when using `noarch: python`, but that might have just been tools built as entry points:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#test-commands

File: recipes/hjson/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
```
Comment (sodre) [inline recipes/hjson/meta.yaml]: Declare any entry-points from upstream/setup.py here:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html?highlight=entry_points#python-entry-points

**Comment by nickpeihl** [comment]
Sorry, I don't have the bandwidth to continue this at the moment.

**Pull request closed without merging**